apps/core/lib/core/pubsub/protocols/fanout.ex

```diff
@@ -45,6 +45,15 @@ defimpl Core.PubSub.Fanout, for: Core.PubSub.UserCreated do
   end
 end
 
+defimpl Core.PubSub.Fanout, for: Core.PubSub.UserUpdated do
+  alias Core.Services.Users
+
+  def fanout(%{item: %{email_changed: true} = user}) do
+    Users.create_reset_token(%{type: :email, email: user.email})
+  end
+  def fanout(_), do: :ignore
```

Comment:
this is a mega-nit, but we usually just return `:ok` for these

Reply:
Ok, done.